Dockerfile.service

```diff
@@ -1,11 +1,45 @@
-# you probably don't want to use this Dockerfile directly,
+# you probably don't want to build this Dockerfile directly,
 # use `make <svc>` instead
 
+#
+# pull deps
+#
+FROM debian:12.8 AS deps
+
+
+# renovate: datasource=github-releases depName=golang-migrate/migrate
+ARG MIGRATE_VERSION=4.18.1
+
+
+RUN apt update && \
+	apt install -y wget
```

Comment:
Just a quick note here. Do you mind adding https://github.com/hadolint/hadolint and pinning versions here?

Reply:
either way I'm not touching it right now, no time and will to debug this right now